lib/matplotlib/text.py

```diff
@@ -1431,11 +1431,9 @@ def __init__(self,
 
         self._draggable = None
 
-    def _get_xy(self, renderer, x, y, s):
-        if isinstance(s, tuple):
-            s1, s2 = s
-        else:
-            s1, s2 = s, s
+    def _get_xy(self, renderer, xy, s):
```

Comment:
While at it: I'm not 100% sure, but isn't this logically
```suggestion
    def _get_xy(self, renderer, xy, xycoords):
```

Reply:
"coords", as it can also be other kinds of coords, but sure.